crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI064.py

```diff
@@ -0,0 +1,16 @@
+from typing import Final, Literal
+
+x: Final[Literal[True]] = True  # PYI064
+y: Final[Literal[None]] = None  # PYI064
+z: Final[Literal[  # PYI064
+    "this is a really long literal, that won't be rendered in the issue text"
+]] = "this is a really long literal, that won't be rendered in the issue text"
+
+# This should be fixable, and marked as safe
+w1: Final[Literal[123]]  # PYI064
+
+# This should be fixable, but marked as unsafe
+w2: Final[Literal[123]] = "random value"  # PYI064
```

Comment:
I think an unsafe autofix is appropriate here (specifically when the assignment value is different from the literal value).

Is autofixing it to `w2: Final = 123` in unsafe mode fair?

Reply:
ummmmmmm this kind of thing doesn't really make any sense, right? I think I'd lean towards not doing an autofix here -- "refuse the temptation to guess"

Reply:
fair enough.

Reply:
done.